### PR TITLE
Add igwn-cmake-macros

### PR DESCRIPTION
**Pull request by duncanmmacleod**
This PR adds a recipe for `igwn-cmake-macros` which is just a renamed version of `igwn-cmake`.

@conda-forge/core, can someone archive the [`igwn-cmake-feedstock`](https://github.com/conda-forge/igwn-cmake-feedstock/), please?

@emaros, please confirm you are happy to maintain this recipe in place of the old one.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by emaros** [comment]
I confirm that I am  happy to maintain this recipe in place of the old one.

**Comment by duncanmmacleod** [comment]
@conda-forge/staged-recipes, this recipe is ready for review, thanks.

@conda-forge/core, can someone archive the old [`igwn-cmake-feedstock`](https://github.com/conda-forge/igwn-cmake-feedstock/), please?

**Comment by duncanmmacleod** [comment]
Ping @conda-forge/staged-recipes, @conda-forge/core.

File: recipes/igwn-cmake-macros/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+build:
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - make  # [not win]
+    - pkg-config
+  run_constrained:
+    - igwn-cmake >9999
```
Comment (xhochy) [inline recipes/igwn-cmake-macros/meta.yaml:21]: Even better
```suggestion
    - igwn-cmake <0a0
```

**Pull request merged**